nautobot_floor_plan/filters.py

```diff
@@ -4,7 +4,7 @@
 
 from nautobot.dcim.models import Location, Rack
 from nautobot.extras.filters import NautobotFilterSet
-from nautobot.utilities.filters import NaturalKeyOrPKMultipleChoiceFilter, SearchFilter, TagFilter
+from nautobot.core.filters import NaturalKeyOrPKMultipleChoiceFilter, SearchFilter, TagFilter
```

Comment:
```suggestion
from nautobot.apps.filters import NaturalKeyOrPKMultipleChoiceFilter, SearchFilter
from nautobot.core.filters import TagFilter
```

Though `TagFilter` may not be necessary per se as the filterset **should** automatically have a `tags` (not `tag`) filter field?